datastore/mongodb/filter_outputs.go

```diff
@@ -9,6 +9,19 @@ import (
 	"go.mongodb.org/mongo-driver/bson"
 )
 
+// GetFilterOutput
+func (c *Client) GetFilterOutput(ctx context.Context, filterID string) (*model.FilterOutput, error) {
+	var filterOutput model.FilterOutput
+
+	coll := c.collections.filterOutputs
+
+	if err := c.conn.Collection(coll.name).FindOne(ctx, bson.M{"id": filterID}, &filterOutput); err != nil {
+		return nil, err
```

Comment:
So here with the errors if you check the example in getFilter:
```
if err = c.conn.Collection(col.name).FindOne(ctx, bson.M{"filter_id": fID}, &f); err != nil {
		err := &er{
			err: errors.Wrap(err, "failed to find filter"),
		}
		if errors.Is(err, mongodb.ErrNoDocumentFound) {
			err.notFound = true
		}
		return nil, err
	}
```
you can see how we're annotating the error with information here which we then use in the handler to determine what to do with it

Reply:
@ChrysmOre  not sure why this is not being marked as outdated, but if you look at https://github.com/ONSdigital/dp-cantabular-filter-flex-api/blob/feature/get-filter-outputs/datastore/mongodb/filter_outputs.go you will find that the changes you have requested have been implemented.

Reply:
Outdate just means the file/code has changed since I made the comment